src/odemis/acq/fastem.py

```diff
@@ -276,6 +277,31 @@ def run(self):
         :raise:
             Exception: If it failed before any single field images were acquired or if acquisition was cancelled.
         """
+        self._beamshift.shift.value = (0, 0)
+
+        # Update the position of the first tile.
+        self._pos_first_tile = self.get_pos_first_tile()
+
+        # Move the stage to the first tile, to ensure the autofocus and image
+        # translation pre-align are done in the correct location.
+        self.move_stage_to_next_tile()
```

Comment:
This is actually super nice as it also solves another bug we have. We had the incorrect stage position in the MD stored as yaml file on the external storage with the megafield. As you call this now before the .subscribe, also this should be solved. Can you add a comment that this also ensures that the correct stage position is stored in the MD on the megafield (and test it?)?